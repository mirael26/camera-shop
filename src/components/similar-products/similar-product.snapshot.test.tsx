import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import { productsMock } from '../../test/mocks';
import SimilarProducts from './similar-products';

jest.mock('../product-card/product-card', () => 'ProductCard');

test('SimilarProducts renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<SimilarProducts/>, {
    initialState: {
      data: { similarProducts: productsMock }
    }
  });
  expect(asFragment()).toMatchSnapshot();
});
