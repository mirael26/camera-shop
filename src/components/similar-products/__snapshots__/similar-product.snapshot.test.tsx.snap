// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimilarProducts renders correctly 1`] = `
<DocumentFragment>
  <div
    class="page-content__section"
  >
    <section
      class="product-similar"
      data-testid="similar-products"
    >
      <div
        class="container"
      >
        <h2
          class="title title--h3"
        >
          Похожие товары
        </h2>
        <div
          class="product-similar__slider"
        >
          <div
            class="product-similar__slider-list"
          >
            <productcard
              product="[object Object]"
            />
            <productcard
              product="[object Object]"
            />
            <productcard
              product="[object Object]"
            />
          </div>
        </div>
      </div>
    </section>
  </div>
</DocumentFragment>
`;
